README.md

```diff
@@ -59,15 +60,33 @@ You should see the following output:
 
 ![mine logs](stdout_progress.png)
 
+### 🌙 Command Line Interface
+
+To list available commands, execute `node bin/cli.js --help`:
+```
+Usage: cli [options]
+
+
+  Options:
+
+    -V, --version                     output the version number
```

Comment:
Yea I thought this was silly. `-V` comes default with commander. Will look into overriding.